dev_guide/builds/build_inputs.adoc

```diff
@@ -458,6 +458,53 @@ section in your *_.gitconfig_* file:
 ----
 ====
 
+[[source-secrets-gitconfig-file-secured]]
+==== .gitconfig File for Secured Git
+
+If your Git server is secured with 2-way SSL and user name with password
+you must add the certificate files to your source build and add references to
+them in the *_.gitconfig_* file:
+
+. Add the *_client.crt_*, *_cacert.crt_*, and *_client.key_* files to the 
+*_/var/run/secrets/openshift.io/source/_* folder in the
+xref:../../dev_guide/application_lifecycle/new_app.adoc#specifying-source-code[application
+source code].
+
+. In the *_.gitconfig_* file for the application, add the `[http]` section
+shown in the following example:
++
```

Comment:
@kalexand-rh My ignorance, perhaps. Is the .gitconfig file for the server, repo. application?
Above you talk about the server and gitconfig; here the git config _for the application_.

Reply:
The git config file is in the repo that holds your application code. The git server is the instance that holds all the git repos. If I'm reading it right, the certificate files are good for the whole server, but you need to put the references to them in each application repo's git config.

Reply:
So it is the gitconfig for the repo where the application is held/resides/is stored. That is what I thought. 
What you have is probably fine.

Reply:
@wzheng1, will you please confirm that the gitconfig file is for the application, and that the gitconfig file is not for the entire Git server?

Reply:
@kalexand-rh  the gitconfig file is for the entire Git server.